### DISCUSSION TITLE
General : is SLiM able to simulate plant breeding ?

### DISCUSSION THREAD

**Discussion by AugustinDesprez** [MesserLab/SLiM#460] [General]
Dear Ben Haller,
Dear all,

Currently working on my PhD studying diversity and haplotypes transmission in plant breeding, I am using markers data on a few thousands of plants covering around two decades (20 generations) of plant breeding. Therefore, I have access to genotypic data of a good amount of founders and their progeny. My questions are not exactly in the same time scale as ecology and evolutionary mechanism, and I know some tools are not always easy to adapt, that's why before diving into SLiM, I need the feedback of users. If I am not wrong, SLiM is able to simulate genome and their evolution, which is the main thing I am interested in.

In order to give a proper meaning to the data that I can observe, (i.e. diversity, effective population size, Haplotype size...), I was wondering if I could use simulations from SliM to answer several questions :
1- What demographic scenario (pop size, number of generations, selection pressure...) is best fitting the data I am observing now, when starting from founders (i.e. 10 to 20 founders) ?
Here SLiM mission could be to start from founders and try many combinations to create the result of 20 years of crosses and see witch model(s) are most probably providing the same data that I am able to observe today.

2- Comparing my data from data created without hard selective pressure (only drift for example), which genomic regions are the farthest from this neutral conditions ? And what may have led to be that far (founder effet, selection pressure, recombinations...) ?
Here I thought SLiM mission could to create genotypes obtained with neutral models starting from known founders.

3- Could SLiM be able to recreate pedigrees with founders and progeny (even if not all founders are genotyped) ?
Then, we could be able to know how actual plants could have been obtained with the least amount of generations, and compare if breeding methods were the most efficient or not (efficient could be in generations as making progress fast is a big issue, but could also be regarding having the highest result with few crosses between plants).

Thank you for reading and I would be pleased that you enlighten me on this subject,
I don't know if here is the best place to ask this type of general questions, feel free to redirect me elsewhere if needed,

Sincerely,

Augustin

**Comment by AugustinDesprez** [comment]
Question asked here : https://groups.google.com/g/slim-discuss which may be a better place for it.

**Comment by gregorgorjanc** [comment]
@AugustinDesprez SLiM is a fantastic software for stochastic simulation in genetics, and could in principle do what you are asking for. I am quite interested in how a SLiM script for a plant breeding programme would look like!

We have developed an R package AlphaSimR dedicated for breeding simulations - see https://cran.r-project.org/web/packages/AlphaSimR/index.html, which is probably more suited to what you are asking for;)

**Comment by bhaller** [comment]
Hi @gregorgorjanc and @AugustinDesprez.  Yes, slim-discuss is a better place for questions like this; let's leave this discussion closed.  Gregor, can you post your reply there, please?  Thanks!

**Comment by gregorgorjanc** [comment]
Hi Augustin,

SLiM is a fantastic software for stochastic simulation in genetics, and
could in principle do what you are asking for. I am quite interested in how
a SLiM script for a plant breeding programme would look like!

We have developed an R package AlphaSimR dedicated for breeding simulations
- see https://cran.r-project.org/web/packages/AlphaSimR/index.html, which
is probably more suited to what you are asking for;)

With regards,
  Gregor
